demonstrations/tutorial_vqt.py

```diff
@@ -0,0 +1,530 @@
+                x = np.kron(x, qml.PauliX.matrix)
+                y = np.kron(y, qml.PauliY.matrix)
+                z = np.kron(z, qml.PauliZ.matrix) 
+            else:
+                x = np.kron(x, np.identity(2))
+                y = np.kron(y, np.identity(2))
+                z = np.kron(z, np.identity(2))
+
+        matrix = np.add(matrix, np.add(x, np.add(y, z)))
+
+    return matrix
+
+ham_matrix = create_hamiltonian_matrix(4, interaction_graph)
+print(ham_matrix)
+
+
+######################################################################
+# With this done, we construct the VQT. We begin by defining some
+# fixed variables that are used throughout the simulation:
+#
+
+
+beta = 2  # beta = 1/T
+nr_qubits = 4
+
+
+######################################################################
+# The first step of the VQT is to create the initial density matrix,
+# :math:`\rho_\theta`. In this tutorial, we let :math:`\rho_\theta` be
+# **factorized**, meaning that it can be written as an uncorrelated tensor
+# product of :math:`4` one-qubit density matrices that are diagonal in
+# the computational basis. The motivation is that in this factorized model, 
+# the number of :math:`\theta_i` parameters needed to describe 
+# :math:`\rho_\theta` scales linearly rather than exponentially with 
+# the number of qubits. For each one-qubit system described by 
+# :math:`\rho_\theta^i`, we have:
+#
+# .. math:: \rho_{\theta}^{i} \ = \ p_i(\theta_i) |0\rangle \langle 0| \ + \ (1 \ - \ p_i(\theta_i))|1\rangle \langle1|
+#
+# From here, all we have to do is define :math:`p_i(\theta_i)`, which we
+# choose to be the sigmoid:
+#
+# .. math:: p_{i}(\theta_{i}) \ = \ \frac{e^{\theta_i}}{e^{\theta_i} \ + \ 1}
+#
+
+
+def sigmoid(x):
+
+    return np.exp(x) / (np.exp(x) + 1)
+
+
+######################################################################
+# This is a natural choice for probability function, as it has a range of
+# :math:`[0, \ 1]`, meaning that we don’t need to restrict the domain of
+# :math:`\theta_i` to some subset of the real numbers. With the probability
+# function defined, we can write a method that gives us the diagonal
+# elements of each one-qubit density matrix, for some parameters
+# :math:`\theta`:
+#
+
+
+def prob_dist(params):
+
+    return [[sigmoid(p), 1 - sigmoid(p)] for p in params]
+
+
+######################################################################
+# With this done, we can move on to defining the ansatz circuit,
+# :math:`U(\phi)`, composed of rotational and coupling layers. The 
+# rotation layer is simply :math:`RX`, :math:`RY`, and :math:`RZ` 
+# gates applied to each qubit. We make use of the ``AngleEmbeddings`` 
+# function, which allows us to easily pass parameters into rotational 
+# layers. 
+#
+
+
+def single_rotation(phi_params, qubits):
+
+    rotations = ["Z", "Y", "X"]
+    for i in range(0, len(rotations)):
+        qml.templates.embeddings.AngleEmbedding(phi_params[i], wires=qubits, rotation=rotations[i])
```

Comment:
All templates are imported into the `templates` package 🙂 
```suggestion
        qml.templates.AngleEmbedding(phi_params[i], wires=qubits, rotation=rotations[i])
```